charts/gie-standalone/README.md

```diff
@@ -0,0 +1,173 @@
+# GIE Standalone Deployment
+## Prerequisite
+### etcd
+```
+# TODO: the link need to be updated after merging to main
+kubectl apply -f https://raw.githubusercontent.com/shirly121/GraphScope/add_gie_deploy/charts/gie-standalone/tools/etcd.yaml
```

Comment:
It's ok to have `kubectl apply -f https://raw.githubusercontent.com/alibaba/GraphScope/add_gie_deploy/charts/gie-standalone/tools/etcd.yaml` before merging.